core/templates/dev/head/player/conversation_skin_embed_directive.html

```diff
@@ -0,0 +1,715 @@
+<script type="text/ng-template" id="skins/ConversationEmbed">
+    <!--
+    Off-screen preview of the next card in order to pre-determine the target
+    height for the card content transition animation.
+    -->
+    <div class="conversation-skin-future-tutor-card" aria-hidden="true">
```

Comment:
In this entire file the indentation should be 2 spaces, similar to other templates.